### DISCUSSION TITLE
Prepared Transactionの訳

### DISCUSSION THREAD

**Discussion by KenichiroTanaka** [pgsql-jp/jpug-doc#2490] [Polls]
現行の訳では混在しており、以下のような状態です。

$find . -name "*.sgml"|xargs grep 準備されたトランザクション|grep -v 機械翻訳|wc -l
      32
$find . -name "*.sgml"|xargs grep プリペアドトランザクション|grep -v 機械翻訳|wc -l
      10

**Comment by koizumistr** [comment]
他のところでもpreparedって（同じ文脈で）使われているはずで、そちらも含めて考えた方がいいのかも、と思いました。
が、話が大きすぎる？

**Reply by KenichiroTanaka** [reply]
違う文脈で使われているものがなければ一緒にしてしまっていいかもです。
違う文脈・・・あるでしょうか。
prepared statementがパッと思いつきましたが、他にあるでしょうか。ざっとpreparedで検索していますが、
他は見当たらず(本当にざっとです)。

**Reply by koizumistr** [reply]
私も"prepared statement"が思いつきました。

が、それ以外すぐには出てこなかったのは私の能力の限界、ということで。

**Comment by KenichiroTanaka** [comment]
個人的には「準備されたトランザクション」=「PREPARE TRANSACTION」と認識するのにハードルが高いためプリペアドトランザクションが良いのでは無いかと思っています。

**Reply by tatsuo-ishii** [reply]
田中さんと同じ理由で「プリペアドトランザクション」（「プリペアドステートメント」）に一票。

**Reply by koizumistr** [reply]
> プリペアドステートメント

statementはおそらく「文」と訳していますので、「プリペアド文」ということになるのではないかと思います。
が、今の話題とはちょっと違ったかもですね。失礼しました。

**Comment by KenichiroTanaka** [comment]
@noborus さん
@tatsuo-ishii さん
@harukat さん
@kitayama さん
後少し、どなたかのご意見or投票をいただきたいところだったのでメンションしました。

**Reply by noborus** [reply]
一応上の方で投票できるようになっていて、投票したのですが...
準備されたトランザクションが連語の訳っぽくないので（準備済みトランザクションの方がまだ良いように思えます）、プリペアドで良いように思います。

**Reply by kitayama** [reply]
config.sgml の メモリの max_prepared_transactions パラメータの説明で <quote>prepared</quote> state を  <quote>プリペアド</quote>状態 と訳しているので、状態の呼称を「プリペアド」としたなら「プリペアドトランザクション」が一致してよいかと思います。

**Comment by KenichiroTanaka** [comment]
みなさま、ありがとうございます。
プリペアドトランザクションで反対意見がなさそうですので、そのように進めます。

**Reply by noborus** [reply]
今変更できるタイミングです。
